### DISCUSSION TITLE
Autocomplete delay

### DISCUSSION THREAD

**Discussion by RGBCube** [helix-editor/helix#8782] [Q&A] [answered]
Is there a way I can turn off the autocomplete delay? Waiting for the popup to pop up is too long when editing code.

Also, what's the keybind to manually trigger the autocomplete?

**Comment by Nintron27** [comment, marked as answer]
This should be configurable with the `idle-timeout` and `completion-trigger-len` settings inside the [editor portion](https://docs.helix-editor.com/configuration.html?highlight=idle-ti#editor-section) of the config. The keybinding to trigger the autocomplete popup is `Ctrl-x`.
